abb_irb6600_support/launch/robot_interface_download_irb6600_225_255.launch

```diff
@@ -0,0 +1,20 @@
+<!--
+  Manipulator specific version of abb_driver's 'robot_interface.launch'.
+
+  Defaults provided for IRB 6600:
+   - J23_coupled = false
```

Comment:
This was original true but after testing this model on the actual robot it was found not to be the case.